tomviz/MainWindow.cxx

```diff
@@ -684,22 +768,20 @@ void MainWindow::registerCustomOperators()
 void MainWindow::registerCustomOperators(const QString& path)
 {
   std::vector<OperatorDescription> operators = findCustomOperators(path);
+
   // Sort so we get a consistent order each time we load
   std::sort(operators.begin(), operators.end(),
             [](const OperatorDescription& op1, const OperatorDescription& op2) {
-              return op1.label.compare(op2.label);
+              return  op1.label < op2.label;
```

Comment:
Any reason for this change?

Reply:
Yep, explained in the first commit. Seems like `compare` should have worked, but it was always returning positive integers, so the sort result was the same even when swapping operands `op1` and `op2`. Plus, `std::sort` takes a comparison function that returns a `bool` rather than an `int` [1].

[1] http://en.cppreference.com/w/cpp/algorithm/sort

Reply:
Ok, thanks for the explanation. I have learn something :smile: